routes/annotations.js

```diff
@@ -98,6 +63,18 @@ exports.updateMutation = function updateMutation(req, res) {
 
 };
 
+exports.createMutationRef = function(req, res) {
```

Comment:
As it turns out, we don't need this either because mutation reference creation is handled through annotation MAGI.